qiskit/circuit/library/__init__.py

```diff
@@ -277,9 +278,6 @@
 
 from .standard_gates import *
 from .templates import *
-from ..barrier import Barrier
```

Comment:
Why are these removed?

Reply:
good catch. Reverted in 8ede589fef1195006273b9f25a3c88473964d3b4